pkg/setting/setting_test.go

```diff
@@ -78,16 +78,6 @@ func TestLoadingSettings(t *testing.T) {
 			So(appliedEnvOverrides, ShouldContain, "GF_SECURITY_ADMIN_PASSWORD=*********")
 		})
 
-		Convey("Should return an error when url is invalid", func() {
-			err := os.Setenv("GF_DATABASE_URL", "postgres.%31://grafana:secret@postgres:5432/grafana")
-			require.NoError(t, err)
-
-			cfg := NewCfg()
-			err = cfg.Load(&CommandLineArgs{HomePath: "../../"})
-
-			So(err, ShouldNotBeNil)
-		})
-
```

Comment:
As part of security fix we are redacting all URLs and not only database one, so it no longer makes sense to do this kind of handling specifically for database URL, I think.